src/services/resources-getter.js

```diff
@@ -52,26 +52,7 @@ function ResourcesGetter(model, options, params) {
   let hasSmartFieldSearch = false;
```

Comment:
When filtering on a belongsTo it crash if it's nit shown inside the table because `getFieldNamesRequested ` should be changed with the new format